src/Cli/dotnet/commands/dotnet-run/RunCommand.cs

```diff
@@ -317,7 +317,7 @@ static ICommand CreateCommandFromRunProperties(ProjectInstance project, RunPrope
                 CommandSpec commandSpec = new(runProperties.RunCommand, runProperties.RunArguments);
 
                 var command = CommandFactoryUsingResolver.Create(commandSpec)
-                    .WorkingDirectory(runProperties.RunWorkingDirectory);
+                    .WorkingDirectory(runProperties.RunWorkingDirectory ?? string.Empty);
```

Comment:
could this previously be null and any concerns about being empty now. Assuming we didn't have explicit null vs. nullorwhitespace checks

Reply:
This was already defined as `string? RunWorkingDirectory` at the top of this file. What changed was `WorkingDirectory` now became part of a nullable-enabled assembly, and its parameter is defined as `string projectDirectory`. That means, previously, if this was passed `null`, it would assign it to `process.StartInfo.WorkingDirectory`. If the decompiled code is to be believed, the default for that property is `""`, so I just left it as that in this situation. The code for it does handle null, and it'll just return `string.Empty` anyway.
![image](https://github.com/user-attachments/assets/8ef71cd0-ca46-46fd-90f9-530162227e6c)

Reply:
Based on what I wrote above and double checking the code, I just changed the `WorkingDirectory()` method in this class to take a `string?` instead without issue.